timesketch/api/v1/resources/searchindex.py

```diff
@@ -126,6 +126,7 @@ def delete(self, searchindex_id):
                 HTTP_STATUS_CODE_FORBIDDEN,
                 '\n'.join(error_strings))
 
+        searchindex.set_status(status='deleted')
         db_session.delete(searchindex)
```

Comment:
Why setting the status when you delete the object next?

Reply:
Do we really want to delete the object from the datastore? Or just mark it as deleted? This is what happens when you delete a sketch. It is not fully deleted from SQL, just marked as such.

Reply:
this is the behavior as it has been so far... but that is a good question, I think we should just rather close the index and mark it as deleted.